report/src/test/kotlin/com/hartwig/actin/report/pdf/tables/molecular/DriverTableFunctionsTest.kt

```diff
@@ -21,11 +22,11 @@ class DriverTableFunctionsTest {
     @Test
     fun `Should correctly group by single event`() {
         val externalTrials = setOf(
-            EventWithExternalTrial("PTEN del", trial1),
-            EventWithExternalTrial("PTEN del", trial2),
-            EventWithExternalTrial("MYC amp", trial2),
-            EventWithExternalTrial("PTEN del", trial3),
-            EventWithExternalTrial("MYC amp", trial3)
+            EventWithExternalTrial("PTEN del", trial1, TEST_ACTIONABLE),
+            EventWithExternalTrial("PTEN del", trial2, TEST_ACTIONABLE),
+            EventWithExternalTrial("MYC amp", trial2, TEST_ACTIONABLE),
+            EventWithExternalTrial("PTEN del", trial3, TEST_ACTIONABLE),
+            EventWithExternalTrial("MYC amp", trial3, TEST_ACTIONABLE)
```

Comment:
This looks a bit odd since the event name is no longer consistent with the actionable... would be solved though if you remove event from that class..